crates/swc_ecma_minifier/tests/benches-full/lodash.js

```diff
@@ -1,11 +1,4 @@
-/**
- * @license
```

Comment:
Not sure if this is expected. I think we need to preserve some kinds of comments, and `@license` is one of them

Reply:
This happens because in original code
```js
/**
 * @license
 * Lodash <https://lodash.com/>
 * Copyright OpenJS Foundation and other contributors <https://openjsf.org/>
 * Released under MIT license <https://lodash.com/license>
 * Based on Underscore.js 1.8.3 <http://underscorejs.org/LICENSE>
 * Copyright Jeremy Ashkenas, DocumentCloud and Investigative Reporters & Editors
 */
;(function() {
```
It's considered as an leading comment for `EmptyStatement`, but that `EmptyStatement` is removed during minify. It's expected in the sense that this is how this PR should work, but we still need to get the comment back somehow.

Reply:
I think we may need to fix the comment removal of minifier first (I guess? Not sure)